<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Videojuego</class>
 <widget class="QMainWindow" name="Videojuego">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1468</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Videojuego</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nivel1Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nivel2Button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="salirButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>481</width>
      <height>131</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>621</width>
      <height>581</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>621</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/OneDrive/Desktop/MenuDeOpciones.PNG);
background-position: center;
background-repeat: no-repeat;
background-size: cover;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>salirButton</zorder>
   <zorder>nivel2Button</zorder>
   <zorder>nivel1Button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1468</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
